Production/test/condorSub/dict_EMJ2016.py

```diff
@@ -0,0 +1,161 @@
+flist = {
+    "scenario": "Summer16v3sig",
+    "samples": [
+        ["PrivateSample.EMJ_2016_mMed-1000_mDark-10_ctau-0p1_unflavored-down"],
```

Comment:
PrivateSample. -> PrivateSamples.
(everywhere)

Reply:
Ah... that's a silly mistake